templates/pages/systemSetup.json

```
@@ -0,0 +1,130 @@
+{
+  "name": "System Setup",
+  "layout": [
+    {
+      "type": "tabs",
+      "content": [
+        {
+          "name": "Users",
+          "widgets": [
+            {
+              "name": "User management",
+              "width": 12,
+              "height": 17,
+              "definition": "userManagement",
+              "configuration": {
+                "pollingTime": 30,
+                "pageSize": 5,
+                "sortColumn": "username",
+                "sortAscending": true
+              }
+            }
+          ]
+        },
+        {
+          "name": "Groups",
+          "widgets": [
+            {
+              "name": "User group management",
+              "width": 12,
+              "height": 19,
+              "definition": "userGroups",
+              "configuration": {
+                "pollingTime": 30,
+                "pageSize": 5,
+                "sortColumn": "name",
+                "sortAscending": true
+              }
+            }
+          ]
+        },
+        {
+          "name": "Tenant Management",
+          "widgets": [
+            {
+              "name": "Tenants management",
+              "width": 12,
+              "height": 21,
+              "definition": "tenants",
+              "configuration": {
+                "pollingTime": 30,
+                "pageSize": 5,
+                "sortColumn": "name",
+                "sortAscending": true
+              }
+            }
+          ]
+        },
+        {
+          "name": "System Health",
+          "widgets": [
+            {
+              "name": "Cluster Status",
+              "width": 12,
+              "height": 25,
+              "definition": "highAvailability"
+            }
+          ]
+        },
+        {
+          "name": "System Logs",
+          "widgets": [
+            {
+              "name": "Resource filter",
+              "width": 12,
+              "height": 3,
+              "definition": "filter",
+              "configuration": {
+                "pollingTime": 10,
+                "filterByBlueprints": true,
+                "filterByDeployments": true,
+                "filterByExecutions": true,
+                "filterByNodes": false,
+                "filterByNodeInstances": false,
+                "filterByExecutionsStatus": false,
+                "filterBySiteName": false,
+                "allowMultipleSelection": false
+              },
```

As it is possible to filter logs by nodes and node instances, we should allow `filterByNodes` and `filterByNodeInstances`. Also, Events/Logs widget supports more than one resource, so `allowMultipleSelection` should be turned on.

```suggestion
              "configuration": {
                "pollingTime": 10,
                "filterByBlueprints": true,
                "filterByDeployments": true,
                "filterByExecutions": true,
                "filterByNodes": true,
                "filterByNodeInstances": true,
                "filterByExecutionsStatus": false,
                "filterBySiteName": false,
                "allowMultipleSelection": true
              },
```